hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -69,30 +75,29 @@ const Orders = () => {
                         </Grid>
                         <Grid item lg={12}>
                             <Grid container spacing={1} direction="row">
-                                {Array(12)
-                                    .fill("order card component")
-                                    .map((str, idx) => (
-                                        <Grid
-                                            item
-                                            lg={3}
-                                            md={4}
-                                            sm={6}
-                                            xs={12}
-                                            key={idx}
-                                        >
-                                            <div
-                                                style={{
-                                                    border: "1px solid black",
-                                                    borderRadius: "5px",
-                                                    padding: "10px",
-                                                    textAlign: "center",
-                                                    backgroundColor: "lightblue",
-                                                }}
-                                            >
-                                                {str}
-                                            </div>
-                                        </Grid>
-                                    ))}
+                                {mockPendingOrders.map((order, idx) => (
+                                    <Grid
+                                        item
+                                        lg={3}
+                                        md={4}
+                                        sm={6}
+                                        xs={12}
+                                        key={idx}
+                                        onClick={() => handleClick(order.id)}
+                                    >
+                                        <OrderCard
+                                            teamId={order.team_id}
```

Comment:
could you rename this prop as "teamCode" and use order.team_code instead. The team code is what participants and admins will only see, team_id is just what's generated as the primary key in our database.